Cargo.toml

```diff
@@ -121,6 +122,7 @@ cmd-monorail = { path = "./cmd/monorail", package = "humility-cmd-monorail" }
 cmd-net = { path = "./cmd/net", package = "humility-cmd-net" }
 cmd-openocd = { path = "./cmd/openocd", package = "humility-cmd-openocd" }
 cmd-pmbus = { path = "./cmd/pmbus", package = "humility-cmd-pmbus" }
+cmd-pmp = { path = "./cmd/pmp", package = "humility-cmd-pmp" }
```

Comment:
What if a RV platform does not support PMP? What would be the behavior of this command there? I think it would hit line 70 in lib.rs and not print any PMP info at all?

Reply:
That should be the case.  The CSR read loops will just exit on the first iterations and it won't try to decode anything.

Reply:
Ack